README.md

```diff
@@ -24,21 +26,28 @@ Create new project on Shopsys Framework easily from this package following one o
 ## Documentation
 For documentation of Shopsys Framework itself see [Shopsys Framework Knowledge Base](https://github.com/shopsys/shopsys/blob/master/docs/index.md).
 
-Documentation of the specific project built on Shopsys Framework should be in [Project Documentation](https://github.com/shopsys/project-base/blob/master/docs/index.md).
+Documentation of this specific Demoshop project should be in [Project Documentation](https://github.com/shopsys/demoshop/blob/master/docs/index.md).
```

Comment:
I dont see point in having this sentence here.

Reply:
sentence changed